app/views/declaration/additionalActors/additional_actors_summary.scala.html

```diff
@@ -59,6 +59,10 @@
         Text(messages("declaration.additionalActors.table.multiple.heading", actors.size))
 }
 
+@headerEmptyContent(message: String) = {
+    <span class="govuk-visually-hidden">@message</span>
+}
```

Comment:
IMHO it is worth extracting this part into a component and re-use it where needed. Maybe could be named `spanVisuallyHidden` ?